glossary-entries.tex

```diff
@@ -54,3 +54,4 @@
 \newacronym{hdd}{HDD}{hard disk drive}
 \newacronym{ecc}{ECC}{error correcting code}
 \newacronym{ascii}{ASCII}{American Standard Code for Information Interchange}
+\newacronym{btfs}{BTFS}{Better File System}
```

Comment:
`btfs` is the BitTorrent file system. I'm guessing you mean `btrfs`, also sometimes called _butter fuss_? :)